CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * PaymentButtons
   * Undeprecate `PayPalCreditButtonColor.BLACK` and `.DARK_BLUE`
   * Undeprecate `PayPalButtonColor.BLUE`, `.BLACK`, and `.SILVER`
+  * Added analytics events
```

Comment:
This is internal functionality not anything the merchant needs to be aware of right? I don't think we need a CHANGELOG if so

Reply:
I think we have tried to add them to the changelog just for our own knowledge of what version certain analytics were released in. Though I think we can be more specific here on what analytics we are now sending.

Reply:
I think iOS went with the changelog. I've updated my PR based on the discussions on iOS PR, to maintain parity.

@jaxdesmarais what kind of specificity are you wanting to see in the changelog?

Reply:
+1 to having a changelog entry so we can track what version we added analytics in for buttons module

Reply:
I think the update you made in [this commit](https://github.com/paypal/paypal-android/pull/245/commits/e079d00d63827867e99a779cc1508e1500b852d5) is in line with the specificity that I was thinking! Thanks for making that change.